ONSdigital/eq-survey-runner
# Forms + Validation

This PR adds some basic form styling along with basic validation using clientside + proof-of-concept remote validation. It also addresses some issues with supporting IE8 and below.
## Changes
- Added [parsleyjs](parsleyjs.org) validation library.
- Added new temporary route at `/validate/` with basic field validation. 
- Added basic form element and button styling (work in progress).
- Added Prismjs syntax highlighting to pattern library.
- Refactored some Sass
- Fixed issue with Gulp task crashing
## How to test
- `npm run compile` to build front-end assets.
- Navigate to `/patterns/forms`.
- Confirm both validation examples are working as expected.
- No need to test IE at this point.
